src/Sarif.Viewer.VisualStudio/SarifSnapshot.cs

```diff
@@ -125,7 +128,20 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         //url = string.Format("http://www.bing.com/search?q={0} {1}", _errors[index].Provider.Name, _errors[index].ErrorCode);
                     }
 
-                    content = Uri.EscapeUriString(url);
+                    if (url != null)
+                    {
+                        content = Uri.EscapeUriString(url);
+                    }
+                }
+                else if (columnName == StandardTableKeyNames.ErrorCodeToolTip)
+                {
+                    var error = _errors[index];
+                    content = error.RuleId + ":" + error.RuleName;
+                }
+                else if (columnName == StandardTableKeyNames.DetailsExpander)
+                {
+                    var error = _errors[index];
+                    content = !string.IsNullOrEmpty(error.FullMessage);
```

Comment:
I understand that the `DetailsExpander` "column" isn't a real column, it's just a Boolean that says whether to show the expander triangle. And I see about where you put `fullMessage` into the `SarifError` object. But I don't see where `fullMessage` becomes the _content_ of the expander.